cypress/views/common.ts

```diff
@@ -0,0 +1,89 @@
+import { DATA_FEDERATION, DATA_SERVICES } from '../constants/common';
+
+export const MCGMSCommon = {
+  visitMCGMSDashboard: () => {
+    cy.log(
+      'navigating to mcg-ms-console operator from the side navigation bar'
+    );
+    cy.clickNavLink([DATA_SERVICES, DATA_FEDERATION]);
+    cy.log('making sure we are routed to proper endpoint');
+    cy.location('pathname').should('eq', '/mcgms/cluster');
+  },
+  visitBucketPolicyList: () => {
+    cy.clickNavLink([DATA_SERVICES, DATA_FEDERATION]);
+    cy.byLegacyTestID('horizontal-link-Buckets').click();
+  },
+};
+
+export const app = {
+  waitForDocumentLoad: () => {
+    cy.document().its('readyState').should('eq', 'complete');
+  },
+  waitForLoad: (timeout: number = 160000) => {
+    cy.get('.co-m-loader', { timeout }).should('not.exist'); // eslint-disable-line cypress/require-data-selectors
```

Comment:
what are these checks for, can we have a log mentioning it?

Reply:
`waitForLoad` should be clear enough... still added log anyway...